tests/python/pants_test/engine/examples/visualizer.py

```diff
@@ -12,13 +12,15 @@
 
 from pants.base.cmd_line_spec_parser import CmdLineSpecParser
 from pants.engine.fs import PathGlobs
-from pants.engine.subsystem.native import Native
 from pants.util import desktop
 from pants.util.contextutil import temporary_file_path
 from pants_test.engine.examples.planners import setup_json_scheduler
-from pants_test.subsystem.subsystem_util import subsystem_instance
+from pants_test.engine.util import init_native
 
 
+# TODO: These aren't tests, so they should be under examples/.
+# Should they use
```

Comment:
This module was completely broken before I fixed it up, using a method that no longer exists.  So nothing is currently exercising this code.  And I don't see any references to `visualizer` anywhere else in the codebase.